test/functions/compare-loose.js

```diff
@@ -1,6 +1,6 @@
 const { test } = require('tap')
 const compareLoose = require('../../functions/compare-loose')
-const SemVer = require('../../classes/SemVer')
+const SemVer = require('../../classes/semver')
```

Comment:
Called it!  :P

Reply:
😄 Stupid macOS and its permissive casing.